server/templates/staff/sidebar.course.html

```diff
@@ -35,6 +35,11 @@
                     </a>
                   </li>
                   <li>
+                    <a href="{{ url_for('.canvas_course', cid=course.id) }}">
+                      <i class="fa fa-address-book {% if (request.endpoint == 'admin.canvas_course' and request.view_args['cid'] == course.id) %}text-blue{% endif %}"></i> bCourses
```

Comment:
If you want a better place - a link under course settings might be good 

![image](https://cloud.githubusercontent.com/assets/882381/20296567/7aa7879e-aac0-11e6-8700-a28aa6623298.png)

It is kind of hidden - but that should be ok